Makefile_packaging.mk

```diff
@@ -385,9 +390,16 @@ sl42_REPOS += $(OPENSUSE_MIRROR)/update/leap/42.3/oss/                 \
 sl15_REPOS += $(OPENSUSE_MIRROR)/update/leap/15.1/oss/            \
 	      $(OPENSUSE_MIRROR)/distribution/leap/15.1/repo/oss/
 
+define install_gpg_key
+	curl -L -f -O "$(1)";                           \
+	if ! sudo rpm --import repomd.xml.key; then   \
+	    cat repomd.xml.key;                       \
```

Comment:
Line continuation lineups in the above 3 lines.

Reply:
These are consistent for the define block which is separate from the other code blocks.

Reply:
394 doesn't line up with 395 and 396 according to the above.  Those are all within the same block of shell code inside the `define` block.

Reply:
Not seeing that alignment issue on the code section above, and current master looks correct in Visual Studio Code.